docs/data/sql_functions_zh.yml

```diff
@@ -712,6 +712,11 @@ conversion:
       特别是匿名的内联数据类型没有可序列化的字符串表示。在这种情况下返回 `NULL`。
 
 collection:
+  - sql: ARRAY_SLICE(array, start_offset, end_offset)
+    table: array.arraySlice(start_offset, end_offset)
+    description: 返回输入数组的子集，从“start_offset”开始到“end_offset”结束。这些偏移量从 1 开始，可以是正数（从数组开头算起）或负数（从数组末尾算起）。如果 'start_offset' 或 'end_offset' 为 0，则将其视为
+      从输入数组的开头开始。如果 'start_offset' 在 'end_offset' 之后或者输入数组为空，它将返回一个空数组。如果“start_offset”或“end_offset”超过数组边界，它们将调整为数组的大小。反之，如果负数 'start_offset' 或 'end_offset'绝对值
+      大于数组的大小，则相应的偏移量将重置为数组的开头，即 1。如果任何输入为 null，则返回 null。
```

Comment:
Looks like you need to rebase to put description of this function in a right place (more function descriptions were added to this file recently)

Reply:
ok